pkg/api/annotations.go

```diff
@@ -399,11 +411,16 @@ func AnnotationTypeScopeResolver() (string, accesscontrol.AttributeScopeResolveF
 			return "", err
 		}
 
-		if annotation.GetType() == annotations.Global {
-			return accesscontrol.ScopeAnnotationsTypeGlobal, nil
+		if annotation.GetType() == annotations.Organization {
+			return accesscontrol.ScopeAnnotationsTypeOrganization, nil
 		} else {
-			return accesscontrol.ScopeAnnotationsTypeLocal, nil
+			return accesscontrol.ScopeAnnotationsTypeDashboard, nil
 		}
 	}
 	return accesscontrol.ScopeAnnotationsProvider.GetResourceScope(""), annotationTypeResolver
 }
+
+func (hs *HTTPServer) canCreateOrganizationAnnotation(c *models.ReqContext) (bool, error) {
+	evaluator := accesscontrol.EvalPermission(accesscontrol.ActionAnnotationsWrite, accesscontrol.ScopeAnnotationsTypeOrganization)
```

Comment:
So far we've been adding separate actions for resource creation, updating and deletion. So we should probable introduce a new `annotations:create` action and use that for this evaluation.

Reply:
Done!